Assets/Tests/PlayModeTests/ViewsTests.cs

```diff
@@ -273,14 +268,724 @@ public async void StartField_AfterDeviceIdChangeWithoutMerge()
+            TestUtility.ValidateRQEQSize(cly, 2, 1);
+            CountlyEventModel view2Pause = cly.Events._eventRepo.Dequeue();
+            TestUtility.ViewEventValidator(view2Pause, 1, 0, 1, TestUtility.BaseViewTestSegmentation(viewNames[1], false, false), viewId2, viewId, null, null, TestUtility.TestTimeMetrics());
+
+            views.ResumeViewWithID(viewId2);
+            TestUtility.ValidateRQEQSize(cly, 2, 0);
+
+            string viewId3 = views.StartView(viewNames[2]);
+            TestUtility.ValidateRQEQSize(cly, 2, 2);
+            CountlyEventModel view2Stop = cly.Events._eventRepo.Dequeue();
+            TestUtility.ViewEventValidator(view2Stop, 1, 0, 0, TestUtility.BaseViewTestSegmentation(viewNames[1], false, false), viewId2, viewId, null, null, TestUtility.TestTimeMetrics());
+
+            CountlyEventModel view3Start = cly.Events._eventRepo.Dequeue();
+            TestUtility.ViewEventValidator(view3Start, 1, 0, null, TestUtility.BaseViewTestSegmentation(viewNames[2], true, false), viewId3, viewId2, null, null, TestUtility.TestTimeMetrics());
+            TestUtility.ValidateRQEQSize(cly, 2, 0);
+
+            Thread.Sleep(1000);
+
+            views.SetGlobalViewSegmentation(testSegmentation);
+            TestUtility.ValidateRQEQSize(cly, 2, 0);
+            views.PauseViewWithID(viewId3);
+            TestUtility.ValidateRQEQSize(cly, 2, 1);
+
+            CountlyEventModel view3Pause = cly.Events._eventRepo.Dequeue();
+            TestUtility.ViewEventValidator(view3Pause, 1, 0, 1, testSegmentation, viewId3, viewId2, null, null, TestUtility.TestTimeMetrics());
+
+            views.ResumeViewWithID(viewId3);
+            TestUtility.ValidateRQEQSize(cly, 2, 0);
+
+            Dictionary<string, object> updatedSegmentation = testSegmentation;
+            updatedSegmentation.Add("another string", "another object");
+            views.UpdateGlobalViewSegmentation(updatedSegmentation);
+            TestUtility.ValidateRQEQSize(cly, 2, 0);
+
+            views.StopViewWithName(viewNames[0]);
+            TestUtility.ValidateRQEQSize(cly, 2, 1);
+            CountlyEventModel view1Stop = cly.Events._eventRepo.Dequeue();
+            TestUtility.ViewEventValidator(view1Stop, 1, 0, 2, updatedSegmentation, viewId, viewId2, null, null, TestUtility.TestTimeMetrics());
+
+            Thread.Sleep(1000);
+
+            views.StopAllViews(null);
+            TestUtility.ValidateRQEQSize(cly, 2, 1);
+            CountlyEventModel view3Stop = cly.Events._eventRepo.Dequeue();
+            TestUtility.ViewEventValidator(view3Stop, 1, 0, 1, updatedSegmentation, viewId3, viewId2, null, null, TestUtility.TestTimeMetrics());
+            TestUtility.ValidateRQEQSize(cly, 2, 0);
+        }
 
         [SetUp]
         [TearDown]
         public void End()
         {
-            Countly.Instance.ClearStorage();
             TestUtility.TestCleanup();
-            Object.DestroyImmediate(Countly.Instance);
```

Comment:
why these removed?

Reply:
public static void TestCleanup(){
    Countly.Instance.ClearStorage();
    UnityEngine.Object.DestroyImmediate(Countly.Instance);
}

this is the function that we want to standardize for each test, which also does this exact same thing. So no need to keep these